docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -139,9 +139,18 @@ Currently there is no option to select an alternative region for storage.
 
 ## Will Teleport be automatically upgraded with each release?
 
-The Teleport control plane is routinely upgraded within 1-2 weeks of a Teleport Core or Enterprise Release.
-New features may not be available to cloud users until compatibility is tested and fixed, and some features may not work in the cloud environment.
-Individual customers may not be upgraded to the latest release if they are running agents that are outside of the supported compatibility of the control plane.
+Teleport Cloud follows the Teleport [Production
+readiness](../../preview/upcoming-releases.mdx) guidelines.
+
+- Teleport Cloud control plane major version upgrades occur after the first
```

Comment:
This is the only place in the entire docs where we refer to the Teleport Cloud _control plane_. If we want to use a new term we should probably define it first.

Alternatively, just say Auth Servers and Proxy Servers.